TestSwitchApi/TestSwitchApi/Models/Response/CandidateTestResponseModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using TestSwitchApi.Models.DataModels;
+using TestSwitchApi.Repositories;
+
+namespace TestSwitchApi.Models.Response
+{
+    public class CandidateTestResponseModel
+    {
+        /*private IEnumerable<CandidateTestModel> _candidateTests;
+        private readonly ICandidateTestsRepo _submissions;*/
```

Comment:
Could you delete this commented out code?

Reply:
deleted commented code